utils/generate_dotnet_channels.js

```diff
@@ -43,6 +42,10 @@ function mapType(type) {
     return 'int';
   if (type === 'ResourceTiming')
     return 'RequestTimingResult';
+  if (type === 'LifecycleEvent')
```

Comment:
these got renamed upstream (in TypeScript) so we need to map them to the previous names to not break users.

Reply:
Let's add a big TOGO: "why do channels use public enums???"